bokehjs/test/models/glyphs/index.coffee

```diff
@@ -1,3 +1,4 @@
 require "./image"
 require "./image_rgba"
 require "./image_url"
+require "./rect"
```

Comment:
I added the newline back. I think GitHub doesn't consider adding a newline changing the code, so it's still showing up here.